nautobot/utilities/utils.py

```diff
@@ -187,6 +188,17 @@ def serialize_object_v2(obj):
     return data
 
 
+def model_to_json(obj, cls=None):
+    """
+    Convenience method to convert object instance to json via a serializer.
+    """
+    if cls:
```

Comment:
1) `cls` is a misleading parameter name, seems like it should be something like `import_string`?
2) I'm also not sure I see the use case here. This is for if you want to serialize it to a dict via some class other than the standard Nautobot serializer class? Is there a specific example you have in mind?

Reply:
1. Makes sense
2. See response on other comment.

Reply:
updated name to dotted_path which is what django calls that parameter. Updated name of function as well.